transcom/mymove
# B-20777 tio queue management

# [INT PR](https://github.com/transcom/mymove/pull/13907)

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A984284&RoomContext=TeamRoom%3A848240&concept=TeamRoom)

## Summary

Adds Queue Management to the TIO queue.
Contains [B-20531](https://github.com/transcom/mymove/pull/13862/)

### How to test

FF Off
1. Access the Office App
2. Login as TIO
3. There should be no Assigned column

FF On
1. Access the Office App
2. Login as TIO
3. There should be an Assigned column
4. Click the dropdown in the assigned column and see the office user you're logged in as
5. Click your name
6. refresh the page
7. Assigned column now has a value
